Modules/binascii.c

```diff
@@ -592,6 +692,511 @@ binascii_b2a_base64_impl(PyObject *module, Py_buffer *data, int newline)
     return PyBytesWriter_FinishWithPointer(writer, ascii_data);
 }
 
+/*[clinic input]
+binascii.a2b_ascii85
+
+    data: ascii_buffer
+    /
+    *
+    fold_spaces: bool = False
+        Allow 'y' as a short form encoding four spaces.
+    wrap: bool = False
+        Expect data to be wrapped in '<~' and '~>' as in Adobe Ascii85.
+    ignore: Py_buffer(c_default="NULL", py_default="b''") = None
+        An optional bytes-like object with input characters to be ignored.
+
+Decode Ascii85 data.
+[clinic start generated code]*/
+
+static PyObject *
+binascii_a2b_ascii85_impl(PyObject *module, Py_buffer *data, int fold_spaces,
+                          int wrap, Py_buffer *ignore)
+/*[clinic end generated code: output=6ab30f2a26d301a1 input=11c60c016d4f334b]*/
+{
+    const unsigned char *ascii_data = data->buf;
+    Py_ssize_t ascii_len = data->len;
+    binascii_state *state = NULL;
+
+    assert(ascii_len >= 0);
+
+    /* Consume Ascii85 prefix and suffix if present. */
+    if (wrap) {
+        if (ascii_len < 2
+            || ascii_data[ascii_len - 2] != BASE85_A85_AFFIX
+            || ascii_data[ascii_len - 1] != BASE85_A85_SUFFIX)
+        {
+            state = get_binascii_state(module);
+            if (state != NULL) {
+                PyErr_SetString(state->Error,
+                                "Ascii85 encoded byte sequences must end with b'~>'");
+            }
+            return NULL;
+        }
+        ascii_len -= 2;
+        if (ascii_len >= 2
+            && ascii_data[0] == BASE85_A85_PREFIX
+            && ascii_data[1] == BASE85_A85_AFFIX) {
```

Comment:
PEP 7: if the condition takes multiple lines, move `{` at a separate line.
```suggestion
            && ascii_data[1] == BASE85_A85_AFFIX)
        {
```

Reply:
Thanks, missed this, will do.